tests/PHPUnit/Fixtures/ManySitesImportedLogs.php

```diff
@@ -238,6 +243,11 @@ private function replayLogFile()
                       '--replay-tracking'           => false);
 
         self::executeLogImporter($logFile, $opts);
+
+        // execute again but this time with different idsite
+        $opts['--idsite'] = 3;
+
+        self::executeLogImporter($logFile, $opts);
```

Comment:
Instead of executing `executeLogImporter` twice in this method,can you add the idSite as a parameter to this method (ie `replayLogFile($idSite)`) and call `replayLogFile` twice. Since executing the log importer can take a bit, I think it's better to only execute it once in these helper methods, and execute it multiple times (indirectly through said helper methods) in the `trackVisits` method. This way it's explicitly clear how many times the log importer is executed.